src/Web/Controllers/AccountController.cs

```diff
@@ -73,11 +86,38 @@ public async Task<IActionResult> SignIn(LoginViewModel model, string returnUrl =
             }
             ViewData["ReturnUrl"] = returnUrl;
 
-            var result = await _signInManager.PasswordSignInAsync(model.Email, model.Password, model.RememberMe, lockoutOnFailure: false);
-            if (result.Succeeded)
+            ApplicationUser applicationUser = new ApplicationUser()
+            {
+                UserName = model.Email
+            };
+            var passwordHash = _passwordHasher.HashPassword(applicationUser, model.Password);
+
+            var deviceContext = new DeviceContext
+            {
+                IPAddress = _contextAccessor.HttpContext.Connection.RemoteIpAddress.ToString()
+            };
+
+            SignInRequest req = new SignInRequest()
```

Comment:
Can you use ```var``` and also the trailing ```()``` is not needed.  Also, can you rename the variable to ```signIn```?